docsrc/sasl/authentication_mechanisms.rst

```diff
@@ -52,30 +67,51 @@ Not sure how to get GSSAPI going? Check out our :ref:`GSSAPI configuration guide
 GSS-SPEGNO
 ----------
 
-.. todo::
-   Content needed here
+.. note::
+   If you have more informations about `GSS-SPEGNO` let us know.
 
 KERBEROS_V4
 -----------
 
-.. todo::
-   Content needed here
+Kerberos is used within Microsoft Active Directory as the preferred Authentication Protocol.
```

Comment:
The Kerberos V4 mech is totally obsolete. Your note here is wrong as M$ AD uses Kerberos V5 (via GSSAPI).
This mech is being deleted from the code base so this section can go away.